kasa/smartstrip.py

```diff
@@ -151,6 +148,20 @@ async def set_alias(self, alias: str) -> None:
         """
         return await super().set_alias(alias)
 
+    def _async_verify_emeter(self) -> None:
+        """Raise an exception if there is not emeter."""
+        if not any(plug.has_emeter for plug in self.children):
+            raise SmartDeviceException("Device has no emeter")
```

Comment:
Not that it makes a difference (as there's no I/O involved here), I don't think there is a need for a separate method for this; the parent's `has_emeter` should work just fine.